lucetc/src/function.rs

```diff
@@ -46,7 +48,9 @@ impl<'a> FuncInfo<'a> {
             let vmctx = self.get_vmctx(func);
             let global_base_value = func.create_global_value(ir::GlobalValueData::Load {
                 base: vmctx,
-                offset: GLOBAL_BASE_OFFSET.into(),
+                offset: (-(std::mem::size_of::<InstanceRuntimeData>() as i32)
```

Comment:
I went back and forth a bit on `-size_of() + offset` vs `offset - size_of()` and I think the former reads a bit more nicely. Not super happy with how it reads either way.